tests/test_external_dataset.py

```diff
@@ -0,0 +1,144 @@
+# Copyright (c) Alibaba, Inc. and its affiliates.
+import unittest
+
+from federatedscope.core.auxiliaries.data_builder import get_data
+from federatedscope.core.auxiliaries.utils import setup_seed, update_logger
+from federatedscope.core.configs.config import global_cfg
+from federatedscope.core.fed_runner import FedRunner
+from federatedscope.core.auxiliaries.worker_builder import get_server_cls, get_client_cls
+
+
+class ExternalDatasetTest(unittest.TestCase):
+    def setUp(self):
+        print(('Testing %s.%s' % (type(self).__name__, self._testMethodName)))
+
+    def set_config_torchvision_dataset(self, cfg):
+        backup_cfg = cfg.clone()
+
+        cfg.use_gpu = True
+        cfg.eval.freq = 10
+        cfg.eval.metrics = ['acc']
+
+        cfg.federate.mode = 'standalone'
+        cfg.federate.local_update_steps = 1
+        cfg.federate.total_round_num = 20
+        cfg.federate.batch_or_epoch = 'epoch'
+        cfg.federate.client_num = 5
+        cfg.federate.sample_client_rate = 0.2
+        cfg.federate.share_local_model = True
+        cfg.federate.online_aggr = True
+
+        cfg.data.root = 'test_data/'
+        cfg.data.type = 'MNIST@torchvision'
+        cfg.data.args = [{'download': True}]
+        cfg.data.splits = [0.6, 0.2, 0.2]
+        cfg.data.batch_size = 10
+        cfg.data.transform = [['ToTensor'],
```

Comment:
I am curious about whether the these args (`transform` and `splitter_args`) can be specified in command line?